tests/nat/runtime/test_session_manager.py

```
@@ -0,0 +1,957 @@
+
+
+class MockWorkflow:
+    """Mock workflow for testing."""
+
+    def __init__(self):
+        self.config = MagicMock(spec=Config)
+        self.input_schema = MockInputSchema
+        self.single_output_schema = MockOutputSchema
+        self.streaming_output_schema = MockOutputSchema
+
+    async def run(self, message, runtime_type=RuntimeTypeEnum.RUN_OR_SERVE):
```

Ruff (0.14.8)

Unused method argument: `message`

(ARG002)